lib/tizenCertificateManager.js

```diff
@@ -209,23 +205,23 @@ module.exports = class TizenCM {
         return{afterYear, issuerName};
     }
 
-    static getTizenDeveloperCA() {
+    getTizenDeveloperCA() {
         console.info('[webide-common-tizentv]tizenCM.getTizenAuthorProfile():');
         return caCertPath;
     }
 
-    static getTizenDistributorProfile(privilegeLevel) {
+    getTizenDistributorProfile(privilegeLevel) {
         console.info('[webide-common-tizentv]tizenCM.getTizenDistributorProfile(): privilegeLevel = ' + privilegeLevel);
-				let distributorCA = '';
-				let distributorCertPath = '';
-        if (privilegeLevel == 'public') {
-            distributorCA = distributorPublicCA;
-            distributorCertPath = distributorPublicSigner;
-        }
-        else if (privilegeLevel == 'partner') {
+        let distributorCA = '';
+        let distributorCertPath = '';
+        if (privilegeLevel == 'partner') {
             distributorCA = distributorPartnerCA;
             distributorCertPath = distributorPartnerSigner;
         }
+        else {
+              distributorCA = distributorPublicCA;
+              distributorCertPath = distributorPublicSigner;
```

Comment:
``` js
return {
    distributorCA: distributorCA,
    distributorCertPath: distributorCertPath,
    distributorPassword: distributorPassword
};
```
Refer to return object, the `distributorPassword` is fixed to 'tizenpkcs12passfordsigner'. Is it right?

Reply:
yes, it is fixed password for tizen distributor